readthedocs/proxito/views/serve.py

```diff
@@ -406,6 +406,18 @@ def _register_broken_link(self, project, version, path, full_path):
             if not project.has_feature(Feature.RECORD_404_PAGE_VIEWS):
                 return
 
+            # This header is set from Cloudflare,
+            # it goes from 0 to 100, 0 being low risk,
+            # and values above 10 are bots/spammers.
+            # https://developers.cloudflare.com/ruleset-engine/rules-language/fields/#dynamic-fields.
+            threat_score = int(self.request.headers.get('X-Cloudflare-Threat-Score', 0))
+            if threat_score > 10:
+                log.info(
```

Comment:
I imagine we'll need to drop this to a `log.debug` at some point, but this is fine for now.